docs/customizing.md

```diff
@@ -180,6 +183,58 @@ Example `pulp_install_plugins`:
          upgrade: true
 ```
 
+### Latest patch release for a minor release of pulpcore
+
+Initial installation:
+
+1. Make sure you are running the latest version of the installer, which installs pulpcore's
+   latest 3.y release.
+1. Confirm that all the latest stable releases of your desired plugins are compatible with
+   pulpcore's latest version , such as by reading the release announcement email thread for
+   pulpcore's latest version, reading the plugins README, or as a last resort, reading their `setup.py`.
```

Comment:
They need to identify the latest compatible branch. Like pulp_file 1.20 and 1.21 could both be compatible with pulpcore 3.18 .